controller/src/command.rs

```diff
@@ -323,6 +323,15 @@ where
 			};
 
 			match args.method.as_str() {
+				"qr" => {
+					if !&args.dest.contains(RESPONSE_EXTENTION) {
+						QrToSlate((&args.dest).into()).put_tx(&slate)?;
+						api.tx_lock_outputs(m, &slate, 0)?;
+						return Ok(());
+					}
+					error!("Unable to generate send qr with {RESPONSE_EXTENTION} included in the path, this is the default reply name added to the file");
```

Comment:
Why we can't have the externsion work in the file path for the send?

Reply:
Here is where we cannot generate a transaction with the name "response_" included.